scripts/create-component/plop-templates-storybook/{{componentName}}.stories.tsx.hbs

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { {{componentName}}, {{componentName}}Props } from '@fluentui/{{packageName}}';
+// import { PartialTheme, ThemeProvider } from '@fluentui/react-theme-provider';
```

Comment:
feel free to postpone the change on `withThemeProvider` decorator, but pls remove this line

Reply:
Updated